Tests/CacheAdvanceTests/CacheHeaderHandleTests.swift

```diff
@@ -85,6 +85,84 @@ final class CacheHeaderHandleTests: XCTestCase {
         XCTAssertEqual(headerHandle2.offsetInFileOfOldestMessage, defaultOffsetInFileOfOldestMessage)
     }
 
+    func test_synchronizeHeaderData_resetsFileWhenFileHeaderCannotBeCreated() throws {
```

Comment:
This test exist to hit this code path, which we weren't hitting after my refactor https://github.com/dfed/CacheAdvance/blob/3ec3fab1d51f25ece5a2c1a0465b979ceb2e234d/Sources/CacheAdvance/CacheHeaderHandle.swift#L117